src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+import duke.command.Command;
+import duke.common.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+
+/**
+ * Duke, a Personal Assistant Chatbot that helps a person to keep track of various things.
+ *
+ * @author Rama Aryasuta Pangestu
+ */

```

Comment:
Great JavaDocs documentation. :+1:
I have also noticed that you have done this consistently across all your classes and methods. Great job!